source/pdk/1.0/pdk_troubleshooting.md

```diff
@@ -5,9 +5,18 @@ canonical: "/pdk/1.0/pdk_troubleshooting.html"
 description: "Troubleshooting Puppet Development Kit, the shortest path to developing better Puppet code."
 ---
 
+{:.concept}
+## PDK not in shell PATH
+ 
+PDK is not automatically added to the PATH in some shells. To fix this, add the PATH to the affected shells:
+ 
+* For zsh on OS X, add the PATH by adding the line `eval '/usr/libexec/path_helper -s'` to to the zsh resource file (`~/.zshrc`).
+ 
+* For Debian, add a symlink to `/usr/local/bin` by running `sudo ln -sv /opt/puppetlabs/bin/pdk /usr/local/bin/` 
+
 
 {:.concept}
-### Windows: Execution policy restrictions
+## Windows: Execution policy restrictions
 
 In some Windows installations, trying to run the `pdk` command causes errors related to execution policy restrictions.
```

Comment:
Unrelated: don't call this "errors" say `In some Windows installations, trying to run the `pdk` command is prohibited by the default execution policy restrictions.`